client/my-sites/stats/modernized-layout-styles.scss

```
@@ -0,0 +1,54 @@
+@import "@automattic/components/src/highlight-cards/variables.scss";
+
+$stats-background-color: #fdfdfd;
+$stats-sections-max-width: 1224px;
+
+// Elements above main layout content per pages
+.stats__section-header,
+.store-stats__section-header,
+.stats-navigation,
+.stats-banner-wrapper {
+	background-color: var(--studio-white);
+}
+
+// Main layout content
+.stats {
+	background-color: $stats-background-color;
+
+	// Ensures vertical padding for certain sections.
+	> .highlight-cards,
+	> .stats__all-time-highlights-section {
+		padding-top: $vertical-margin;
+		padding-bottom: $vertical-margin;
+	}
+
+	// Ensures horizontal padding for all sections.
+	@media ( min-width: 660px ) {
+		> * {
+			padding: 0 max(calc(50% - #{$stats-sections-max-width / 2}), 32px);
```

```suggestion
			padding: 0 max(calc(50% - #{math.div($stats-sections-max-width, 2)}), 32px);
```

Could we tweak this to use `math.div` instead of `number / number`? [This PR](https://github.com/Automattic/wp-calypso/pull/70229) is working on applying this exact change.